.github/workflows/test_migrate.yml

```diff
@@ -40,3 +46,31 @@ jobs:
           POSTGRES_DB: "postgres"
           POSTGRES_USER: postgres
           POSTGRES_PASSWORD: "postgres"
+
+  publish_python_package:
+    name: Publish python package
+
+    runs-on: ubuntu-latest
+
+    needs: [test_job]
+    if: ${{ startsWith(github.ref, 'refs/tags/') && always() && !cancelled() && !contains(needs.*.result, 'failure') }}
+
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+
+      - name: Install poetry
+        run:
+          curl -sSL https://install.python-poetry.org | python3 -
+
+      - name: Build sdist static artifact
+        run: |
+          poetry install
+          poetry build
+      - name: Publish
+        run: |
+          poetry publish --username ${{ secrets.PYPI_USERNAME }} --password ${{ secrets.PYPI_PASSWORD }}
```

Comment:
@dw - This require us to add `PYPI_USERNAME` and `PYPI_PASSWORD` keys to the Action secrets. I can't inject these myself since I don't have access to the Github Settings - so will follow up via email to add.